Makefile

```diff
@@ -588,56 +588,24 @@ release:
 ###                        Compile Solidity Contracts                       ###
 ###############################################################################
 
-CONTRACTS_DIR := contracts
-COMPILED_DIR := $(CONTRACTS_DIR)/compiled_contracts
-TMP := tmp
-TMP_CONTRACTS := $(TMP)/contracts
-TMP_COMPILED := $(TMP)/compiled.json
-TMP_JSON := $(TMP)/tmp.json
-
-# Compile and format solidity contracts for the erc20 module. Also install
-# openzeppeling as the contracts are build on top of openzeppelin templates.
-contracts-compile: contracts-clean openzeppelin create-contracts-json
-
-# Install openzeppelin solidity contracts
-openzeppelin:
-	@echo "Importing openzeppelin contracts..."
-	@cd $(CONTRACTS_DIR) && \
-	 npm install && \
-	 mv node_modules $(TMP) && \
-	 mv $(TMP)/@openzeppelin . && \
-	 rm -rf $(TMP)
-
-# Clean tmp files
+# Clean up the contracts directory, install the necessary dependencies
+# and then compile the solidity contracts found in the Evmos repository.
```

Comment:
the repo is obvious, maybe is better to specify the folder from which contracts are compiled.

Reply:
it's compiling contracts from all folders? 🤔